src/Chapter01.Tests/Listing01.02B.Tests.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
 
-namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter01.Listing01_06.Tests
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter01.Listing01_02B.Tests
```

Comment:
Do we want to merge this in before running the listing manager or wait for the word docs to get updated?

Reply:
I don't know what the order of operations is.